### DISCUSSION TITLE
Semantic release v.s. monorepo

### DISCUSSION THREAD

**Discussion by ylemkimon** [KaTeX/KaTeX#2517] [General]
Currently, there is no reliable solution to enable `semantic-release` on the monorepo, so we should decide on which.

Some thoughts:
- Semantic release
  * Fully automated release process
- Monorepo
  * Build the font and code together
  * We can modularize KaTeX
  * Separate packages for `contrib` modules and website

**Comment by k4b7** [comment]
I haven't tried https://github.com/pmowrer/semantic-release-monorepo, but it looks like it might allow us to do both.
